nvflare/private/fed/client/communicator.py

```diff
@@ -367,6 +367,31 @@ def send_heartbeat(self, servers, task_name, token, ssid, client_name, engine: C
                 self.logger.info(f"Failed to send heartbeat. Will try again. Exception: {secure_format_exception(e)}")
                 time.sleep(5)
 
+    def send_job_heartbeat(self, token, ssid, client_name, fl_ctx: FLContext):
```

Comment:
No need for this if we simply send aux msg from client_runner.

Reply:
Removed. Changed to use aux_message sending.